src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,26 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import './TodoInfo.scss';
+
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
```

Comment:
You can use destructuring in one line, cuz we have only 3 values we get from user
`const { title, completed, user } = user`